modules/library/src/main/scala/zio/elasticsearch/aggregation/Aggregations.scala

```diff
@@ -210,6 +210,48 @@ private[elasticsearch] final case class Multiple(aggregations: Chunk[SingleElast
     aggregations.map(_.toJson).reduce(_ merge _)
 }
 
+sealed trait PercentilesAggregation
+    extends SingleElasticAggregation
+    with HasMissing[PercentilesAggregation]
+    with WithAgg {
+
+  /**
+   * Sets the `percents` parameter for the [[zio.elasticsearch.aggregation.PercentilesAggregation]].
+   *
+   * @param percents
+   *   a array of percentiles to be calculated for [[zio.elasticsearch.aggregation.PercentilesAggregation]]
+   * @return
+   *   an instance of the [[zio.elasticsearch.aggregation.PercentilesAggregation]] enriched with the `percents`
+   *   parameter.
+   */
+  def percents(percent: Double, percents: Double*): PercentilesAggregation
+}
+
+private[elasticsearch] final case class Percentiles(
+  name: String,
+  field: String,
+  percents: Chunk[Double],
+  missing: Option[Double]
+) extends PercentilesAggregation { self =>
+
+  def missing(value: Double): PercentilesAggregation =
+    self.copy(missing = Some(value))
+
+  def withAgg(agg: SingleElasticAggregation): MultipleAggregations =
+    multipleAggregations.aggregations(self, agg)
+
+  def percents(percent: Double, percents: Double*): PercentilesAggregation =
```

Comment:
Do we have any specific reason why we named this `percents` if the aggregation is `percentiles`?

Reply:
In elastic api the parameter is named `percents`. But i can change the name to `percentiles` if it seems more intuitive.

Reply:
Agreed to leave this named `percents`